WooCommerce/Classes/ViewRelated/Dashboard/Settings/Beta features/BetaFeaturesViewController.swift

```diff
@@ -83,7 +83,9 @@ private extension BetaFeaturesViewController {
     ///
     func configureSections() {
         // This is empty because there aren't any ongoing experiments
```

Comment:
Should we remove this comment?

Reply:
yup! removed here: https://github.com/woocommerce/woocommerce-ios/pull/2659/commits/ec1bbe522374ef9859dd9205f2585484d560091c